operations/system_layout_service/Add_Liquid-Cooled_Cabinets_To_SLS.md

```diff
@@ -4,44 +4,49 @@ This procedure adds one or more liquid-cooled cabinets and associated CDU manage
 
 **`NOTE`** This procedure is intended to be used in conjunction with the top level [Add additional Liquid-Cooled Cabinets to a System](../node_management/Add_additional_Liquid-Cooled_Cabinets_to_a_System.md) procedure.
 
+**`NOTE`** This procedure will only add the liquid-cooled hardware present in an EX2500 cabinet with a single liquid-cooled chassis and one air-cooled chassis.
+
 ## Prerequisites
 
--   The Cray command line interface \(CLI\) tool is initialized and configured on the system.
--   Collect information for the liquid-cooled cabinets being added to the system. For each cabinet collect:
-    * Cabinet component name (xname) (for example `x1004`)
-    * Hardware Management Network (HMN) VLAN ID configured on the CEC (for example `3004`)
-    * Node Management Network (NMN) VLAN ID configured on the CEC (for example `2004`)
-    * Starting compute node ID (NID) (for example `2025`)
-    * Cabinet Type: Mountain (8 chassis) or Hill (2 chassis)
+- The Cray command line interface \(CLI\) tool is initialized and configured on the system.
+- Collect information for the liquid-cooled cabinets being added to the system. For each cabinet collect:
+  - Cabinet component name (xname) (for example `x1004`)
+  - Hardware Management Network (HMN) VLAN ID configured on the CEC (for example `3004`)
+  - Node Management Network (NMN) VLAN ID configured on the CEC (for example `2004`)
+  - Starting compute node ID (NID) (for example `2025`)
+  - Cabinet Type: Mountain (8 chassis) or Hill (2 chassis)
 
--   Collect information for the CDU switches (if any) being added to the system. For each CDU management switch, collect:
-    - CDU switch component name (xname) (for example `d1w1`)
-    - CDU switch brand (for example Dell or Aruba)
-    - CDU switch alias (for example `sw-cdu-004`)
+- Collect information for the CDU switches (if any) being added to the system. For each CDU management switch, collect:
+  - CDU switch component name (xname) (for example `d1w1`)
+  - CDU switch brand (for example Dell or Aruba)
+  - CDU switch alias (for example `sw-cdu-004`)
 
 ## Procedure
 
-1.  Perform an SLS dump state operation:
+1. (`ncn-m#`) Perform an SLS dump state operation:
```

Comment:
Does this procedure need to be run on a master NCN? Or would it work on any NCN with the CLI and docs installed?